src/js/index.js

```diff
@@ -76,29 +76,58 @@ class FHCookieGuard {
         });
     }
 
-    /**
-     * @private
-     */
     _onAcceptCookiesClick() {
-        this._setCookie(1);
+        this._setCookieConsentValue(this.options.cookieName, 1);
```

Comment:
`this.options.cookieName` niet meer meegeven de functie accepteert nu alleen nog een value